roachtest: django failed

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3667540&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3667540&tab=artifacts#/django) on release-21.1 @ [6988edf0505301f9ee52c50d2b88ef5e6999d214](https://github.com/cockroachdb/cockroach/commits/6988edf0505301f9ee52c50d2b88ef5e6999d214):


```
The test failed on branch=release-21.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:204,django.go:216,test_runner.go:733: 
		Tests run on Cockroach v21.1.11-65-g6988edf050
		Tests run against django cockroach-3.2.x
		9027 Total Tests Run
		9026 tests passed
		1 test failed
		614 tests skipped
		2 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: migrations.test_operations.OperationTests.test_remove_field_m2m_with_through (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

It's caused by this error:

```
09:18:14 orm_helpers.go:205: Tests run on Cockroach v21.1.11-65-g6988edf050
Tests run against django cockroach-3.2.x
9027 Total Tests Run
9026 tests passed
1 test failed
614 tests skipped
2 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: migrations.test_operations.OperationTests.test_remove_field_m2m_with_through (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts
```

Will add the failed test into the ignore list of 21.1.

--- a/pkg/cmd/roachtest/django_blocklist.go
+++ b/pkg/cmd/roachtest/django_blocklist.go
@@ -181,8 +181,9 @@ var djangoIgnoreList21_2 = blocklist{
 }
 
 var djangoIgnoreList21_1 = blocklist{
-	"schema.tests.SchemaTests.test_alter_field_db_collation":          "will be fixed in django-cockroachdb v3.2.2",
-	"schema.tests.SchemaTests.test_alter_field_type_and_db_collation": "will be fixed in django-cockroachdb v3.2.2",
+	"migrations.test_operations.OperationTests.test_remove_field_m2m_with_through": "will be fixed in django-cockroachdb v3.2.2",
+	"schema.tests.SchemaTests.test_alter_field_db_collation":                       "will be fixed in django-cockroachdb v3.2.2",
+	"schema.tests.SchemaTests.test_alter_field_type_and_db_collation":              "will be fixed in django-cockroachdb v3.2.2",
 }
 
 var djangoIgnoreList20_2 = blocklist{

